api-scanner/src/test/java/net/corda/plugins/AnnotatedClassTest.java

```diff
@@ -44,9 +44,18 @@ public void testAnnotatedClass() throws IOException {
 
         Path api = pathOf(testProjectDir, "build", "api", "annotated-class.txt");
         assertThat(api).isRegularFile();
-        assertThat(Files.readAllLines(api)).containsOnlyOnce(
-            "@net.corda.annotation.AlsoInherited @net.corda.annotation.IsInherited @net.corda.annotation.NotInherited public class net.corda.example.HasInheritedAnnotation extends java.lang.Object",
-            "@net.corda.annotation.AlsoInherited @net.corda.annotation.IsInherited public class net.corda.example.InheritingAnnotations extends net.corda.example.HasInheritedAnnotation"
-        );
+        assertEquals(
```

Comment:
`assertj` has some rather nice matchers for `List` object so that we don't need to match the entire file:
```java
assertThat(Files.readAllLines(api, UTF_8))
    .containsSequence(
        "@AlsoInherited",
        "@IsInherited",
        "@NotInherited",
        "public class net.corda.example.HasInheritedAnnotation extends java.lang.Object")
    .containsSequence(
        "@AlsoInherited",
        "@IsInherited",
        "public class net.corda.example.InheritingAnnotations extends net.corda.example.HasInheritedAnnotation");
```